otherlibs/unix/unix.mli

```diff
@@ -456,13 +474,13 @@ val isatty : file_descr -> bool
 module LargeFile :
   sig
     val lseek : file_descr -> int64 -> seek_command -> int64
-    (** See {!Unix.lseek}. *)
+    (** See {!lseek}. *)
```

Comment:
This is unfortunately the three cases where using the unqualified label doesn't work: {!lseek} is resolved to Unix(Labels).LargeFile.lseek here. Thus the function points to itself which is not really useful.

Reply:
Thanks. I'll see if a hack can be added for this case (and any other similar ones). Hopefully it will be relatively simple.

Reply:
If there is no simple fix in sight, we can temporarily remove the reference.